.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+### Expected behavior
+
+### Actual behavior
+
+### Steps to reproduce
+
+*Please use markdown syntax, especially for Backtraces and such!*
+
+### program versions
+
+    - operating system
+    - neomutt
```

Comment:
@flatcap but then, shouldn't we anyways encourage to update their version? (or at least test if the old version fixes the issue?)